src/_help/enhance-documentation-content/topics.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Topics
+---
+
+- TOC
+{:toc}
+
+Topics are an exclusive custom property that we've created to allow you to add extended content to your API documentation. They enable you to add context, additional information, or even tutorial elements to guide your readers in using your API.
```

Comment:
```suggestion
`x-topics` is a vendor-specific property that we've created to allow you to add extended content to your API documentation. They enable you to add context, additional information, or even tutorial elements to guide your readers in using your API.
```

Reply:
For my own curiosity: why "vendor-specific"? What is a vendor?